source/clear-linux/concepts/mixer-about.rst

```diff
@@ -0,0 +1,16 @@
+.. _mixer-about:
+
+Mixer
+#######
+
+Mixing refers to composing a custom, versioned image of |CLOSIA| for a specific use case. While upstream |CL| provides options to install bundles for various capabilities, some developers and OSVs may wish to either augment the operating system with functionality from their own packages or modify the structure of current bundles. Mixing offers a method to add capabilities while retaining the version consistency and controlled update stream of |CL| by creating a new distributable image.
```

Comment:
"while retaining the version consistency and controlled update stream".  

I need help deciphering what you mean by this.

Reply:
:-) "Controlled update stream" was directly from the second to last paragraph in the software update doc: https://clearlinux.org/documentation/clear-linux/concepts/swupd-about. I took it to mean using the mixer would allow customization while remaining firmly in the Clear Linux update stream. "Version consistency" was my own addition in that I also understood the mixer to create an image with a base Clear Linux version and a mix version appended. Since it isn't clear, I'd be happy for suggestions. Somewhat related, an example final version number on the Use Mixer Tool would be helpful.

Reply:
@intelkevinputnam.  I understand what you're aiming at, yet I struggled with the phrase too. 
Suggestion: "Mixing offers a method to add capabilities while reaping the benefits of current development the |CL| upstream. As managers of their custom flavor of |CL| , developers choose whether to pull in upstream changes, selectively or automatically. In effect, they independently manage their new distributable image"

Reply:
@bktan8 Can you let me know if this hybrid of @mvincerx and my versions captures the intent of the mixer? "Mixing offers a method to add capabilities while retaining the ability to stay up to date with an upstream version of |CL| and independently manage their new distributable image."

Reply:
@intelkevinputnam I like your condensed revision. LGTM.

Reply:
I actually prefer to not say image, update stream is preferred for me as the image is a secondary task and may never even be needed.

Reply:
@bryteise. Excellent point. Thanks for your review. 
@intelkevinputnam. Do you mind making one more revision of that final line to align with this?

Reply:
@mvincerx @bryteise Final version? "Mixing offers a method to add capabilities while retaining the ability to stay up to date with an upstream version of |CL|."

Reply:
LGTM. @bryteise Any objections?

Reply:
+1 here